lib/kitchen/directions/bake_chapter_introductions/v2.rb

```diff
@@ -65,6 +65,7 @@ def v1_introduction_order(introduction_page:, chapter_intro_html:, title:)
         "> :not([data-type='metadata']):not(.splash):not(.has-splash)"
       ).cut
 
+      title.id = "#{introduction_page.id}_titlecreatedbycookbook"
```

Comment:
@philschatz I guess there is a need to set the title id also in the rest of the versions of introduction order below (`v2_introduction_order`, `v3_introduction_order`) or maybe another better option is to try to set the id in the `bake_title` method definition in line 123.